pom.xml

```diff
@@ -21,37 +21,37 @@
 
   <properties>
     <!--Java & Junit-->
+    <common-web-java.version>1.5.0</common-web-java.version>
+    <home.url></home.url>
     <java-session-handler.version>2.2.0</java-session-handler.version>
     <java.version>1.8</java.version>
+    <!--    These versions are earlier than in the Parent POM-->
     <junit-jupiter-engine.version>5.5.2</junit-jupiter-engine.version>
     <junit-platform-surefire-provider.version>1.2.0</junit-platform-surefire-provider.version>
-    <!--    These versions are earlier than in the Parent POM-->
+    <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+    <maven.compiler.source>${java.version}</maven.compiler.source>
+    <maven.compiler.target>${java.version}</maven.compiler.target>
     <mockito-junit-jupiter.version>3.1.0</mockito-junit-jupiter.version>
-    <structured-logging.version>1.5.0-rc2</structured-logging.version>
-    <thymeleaf-extras-java8time.version>3.0.1.RELEASE</thymeleaf-extras-java8time.version>
-    <thymeleaf-layout-dialect.version>2.3.0</thymeleaf-layout-dialect.version>
-    <web-security-java.version>1.3.8</web-security-java.version>
     <oauth-signin-java.version>0.2.2</oauth-signin-java.version>
-    <common-web-java.version>1.1.0-rc1</common-web-java.version>
-    <home.url></home.url>
-    <!--Spring -->
     <spring-boot-dependencies.version>2.2.0.RELEASE</spring-boot-dependencies.version>
-    <spring-test.version>5.1.3.RELEASE</spring-test.version>
+    <!--Spring -->
     <spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
+    <spring-test.version>5.1.3.RELEASE</spring-test.version>
```

Comment:
Do we need to include the spring test version here ? Wouldn't it be a managed dependency for spring-boot ?

Reply:
Its been a long time since we made this change. So I can't remember exactly why we did it.
I think we specifically wanted to avoid some version or jump to a version ahead of the managed version so that we could use some specific feature.